phone/src/apps/messages/hooks/useMessageActions.ts

```diff
@@ -89,6 +94,8 @@ export const useMessageActions = (): MessageActionProps => {
     (messageDto: Message) => {
       if (messageLoading !== 'hasValue') return;
 
+      if (conversationId !== messageDto.conversation_id) return;
```

Comment:
We already handle this in [useMessageService](https://github.com/project-error/npwd/blob/develop/phone/src/apps/messages/hooks/useMessageService.ts#L29)

Reply:
That's only used for sending messages, while this is for receiving messages.